apps/authentications/src/app/controllers/authentications.controller.v2.ts

```diff
@@ -48,4 +53,29 @@ export class AuthenticationControllerV2 {
       password
     );
   }
+
+  @ApiResponse({
+    status: 400,
+    description: 'will show if some of header is missing',
+  })
+  @ApiOkResponse({
+    status: 201,
+    type: CheckingResponseV2,
+  })
+  @ApiBody({
+    type: CheckEmailExistDto,
+  })
+  @Post('exists/email')
+  @HttpCode(200)
+  async checkEmailExists(
+    @Req() req: HeadersRequest,
```

Comment:
```suggestion
    @Req() req: HeadersRequest,
```
ตรงนี้ได้ใช้ไหมครับถ้าไม่ได้ใช้ก็ ไม่จำเป็นต้องเรียก